src/stores/AppStore.ts

```diff
@@ -1303,12 +1303,40 @@ export class AppStore {
                 await this.loadDefaultFiles();
                 this.setCursorFrozen(this.preferenceStore.isCursorFrozen);
                 this.updateASTColors();
+                if (this.preferenceStore.checkNewRelease) {
+                    this.checkNewRelease();
+                }
             } catch (err) {
                 console.error(err);
             }
         }
     };
 
+    private checkNewRelease = () => {
+        fetch("https://api.github.com/repos/CARTAvis/carta/releases", {headers: {Accept: "application/vnd.github+json"}})
```

Comment:
I suggest you use `axios` instead, and also make this an `async` function so we can use `await`

Reply:
changed to `axios`